ViAn/Video/zoomer.cpp

```diff
@@ -164,6 +164,23 @@ QSize Zoomer::get_viewport_size() const {
     return m_viewport_size;
 }
 
+cv::Rect Zoomer::get_frame_rect() const {
+    return m_frame_rect;
+}
+
+/**
+ * @brief Zoomer::get_info_string
+ * @return easy to read formatted string with the current settings. used for debugging purposes
+ */
+std::string Zoomer::get_info_string() const {
```

Comment:
I guess this can be nice to have while developing, but should probably eventually be removed.

Reply:
It really was. Not needed anymore though, simply missed it